.postman/collection.json

```diff
@@ -3807,22 +4440,38 @@
 									}
 								},
 								"url": {
-									"raw": "{{host}}/api/lectures/{{lecture_id}}",
+									"raw": "{{host}}/api/lectures/6fae868f-c214-440d-a03a-a20d1881f3be",
 									"host": [
 										"{{host}}"
 									],
 									"path": [
 										"api",
 										"lectures",
-										"{{lecture_id}}"
+										"6fae868f-c214-440d-a03a-a20d1881f3be"
```

Comment:
I think it should be a dynamic `lecture_id` here.

Reply:
I was testing and forgot to change it back 😅